.github/workflows/ci.yml

```diff
@@ -139,9 +142,43 @@ jobs:
           name: spectacular-package
           path: dist/packages/spectacular
           if-no-files-found: warn
+      - name: 'Spectacular: Upload docs build artifact'
+        uses: actions/upload-artifact@v2
+        with:
+          name: ${{ env.spectacular-docs-package-name }}
+          path: dist/packages/spectacular-docs
+          if-no-files-found: warn
+
+  deploy-docs:
+    name: Deploy docs to GitHub Pages
+    # if: >
```

Comment:
Can we clean up this comment?

Reply:
Enabling this again so that we only deploy to GitHub Pages after a merge to `main` or a manual CI run on `main`.